.travis.yml

```diff
@@ -23,6 +21,10 @@ matrix:
     - python: 3.5
       env: TOX_ENV=quality
```

Comment:
Maybe we can move it to the `env` section and remove the `matrix` section altogether.